lib/basedriver/image-element.js

```diff
@@ -227,6 +238,15 @@ class ImageElement {
         return imgEl.location;
       case 'getElementRect':
         return imgEl.rect;
+      case 'getAttribute':
+        // /wd/hub/session/:sessionId/element/:elementId/attribute/:name
+        // /wd/hub/session/:sessionId/element/:elementId/attribute/visual should retun the visual data
+        // e.g. ["content-desc","appium-image-element-xxxxx","xxxxx"], ["visual","appium-image-element-xxxxx","xxxxx"]
+        if (args[0] === 'visual') {
+          return imgEl.matchedImage;
+        } else {
```

Comment:
pretty sure this else block is not necessary (without a `break` or `return`, the `default` case will also be executed)

Reply:
Yes, but the lint claimed about this point.
I compared adding an ignoring the lint message vs adding this else block.

Which is safer in JS world? (I thought following the lint was safer for now.)